cpu/kinetis_common/spi.c

```diff
@@ -22,6 +22,10 @@
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
+#ifndef KINETIS_SPI_USE_HW_CS
+#define KINETIS_SPI_USE_HW_CS   0
+#endif
+
```

Comment:
@PeterKietzmann  No, it is also necessary for the lines https://github.com/jfischer-phytec-iot/RIOT/commit/ba9db2d147ac99fe0efaa9e6d7d924708f393e52#diff-1cbad4f25cbb4ed1edc201509a26b427R346 and https://github.com/jfischer-phytec-iot/RIOT/commit/ba9db2d147ac99fe0efaa9e6d7d924708f393e52#diff-1cbad4f25cbb4ed1edc201509a26b427R764.